src/styles/index.scss

```diff
@@ -1,3 +1,126 @@
+@import './variebles';
+
 body {
   margin: 0;
 }
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  &__img {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__name {
+    margin: 0 0 4px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    font-family: Roboto, sans-serif;
```

Comment:
The `font-family` property is repeated in multiple classes. Consider creating a mixin or a class for this common property.